qutebrowser/utils/docutils.py

```diff
@@ -98,6 +98,16 @@ def __init__(self, func):
             self.State.arg_inside: self._parse_arg_inside,
             self.State.misc: self._skip,
         }
+        if doc is None:
+            if sys.flags.optimize < 2:
+                log.commands.warning(
+                    "Function {}.{}() from {} has no docstring".format(
+                        inspect.getmodule(func).__name__,
+                        func.__name__,
```

Comment:
You should be able to simply do [`utils.qualname(func)`](https://github.com/The-Compiler/qutebrowser/blob/v0.8.2/qutebrowser/utils/utils.py#L660-L685) here.